source/_components/zha.markdown

```diff
@@ -26,6 +26,7 @@ There is currently support for the following device types within Home Assistant:
 Known working ZigBee radios:
 
 - Nortek/GoControl Z-Wave & Zigbee USB Adaptor - Model HUSBZB-1
+- XBee Series 2C
```

Comment:
I wonder if it'd be useful to include the required `radio_type` in these examples

```
 - XBee Series 2C (`radio_type: xbee`)
```

Reply:
I would not harm, that's for sure.